stripe/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -91,6 +91,13 @@ open class StripeEditText @JvmOverloads constructor(
         listenForDeleteEmpty()
         determineDefaultErrorColor()
         cachedColorStateList = textColors
+
+        // This will initialize a listener that calls the internal listeners then the external one
+        super.setOnFocusChangeListener { view, hasFocus ->
+            internalFocusChangeListeners.forEach {
+                it.onFocusChange(view, hasFocus)
+            }
+        }
```

Comment:
Can we just call `setOnFocusChangeListener(null)`?

Reply:
That's funny I had that and changed it because I didn't like it.  I can change it back.

Reply:
It's definitely a bit awkward either way.